Resources/doc/10-web-token.rst

```diff
@@ -0,0 +1,18 @@
+Using Web-Token Feature
+=======================
+
+For v2.xx, this bundle supports the Web-Token Framework to ease the
+use of encrypted tokens and key rotations.
+
+.. note::
+
+    This feature is only available woth PHP 8.1+ and Symfony 6.1+.
+
+
+#. Dependency Installation:
+
+#. Keys and Keysets Creation:
+
+#. BundleConfiguration:
+
+#. Key Rotation:
```

Comment:
Help welcome to fill the docs :)

Reply:
I will write a few words about this feature. I'm just waiting for last minute changes before finishing this part.